codechefcli/problems.py

```diff
@@ -306,6 +306,45 @@ def print_problem_tags(tags):
         print(SERVER_DOWN_MSG)
 
 
+def get_ratings(country, institution, institution_type, lines, page):
+    """
+    :desc: displays the ratings of users. Result can be filtered according to
+    the country, institution, institution_type and sets. `line` decide the
+    number of lines to be shown. `page` tells which page of the result is to be shown
+    :param: values of all filters, lines to be displayed, page no. to be displayed
+    """
+    session = get_session()
+    url = BASE_URL + '/api/ratings/all?sortBy=global_rank&order=asc&page='
+    url += str((page or 1)) + '&itemsPerPage=' + str((lines or 20))
```

Comment:
Use `params` argument. See [here](https://github.com/parascoder1/codechef-cli/blob/2708dc6f08b7b0e3e1732890dc0e664b703e3288/codechefcli/problems.py#L397).  
And, define default values for `lines` (eg: 20) in `constants.py`.

Reply:
1)sir params wale tarike se aisa urls kaise banaye..
_https://www.codechef.com/api/ratings/all?sortBy=global_rank&order=asc&page=1&itemsPerPage=20&filterBy=Country=india;Institution=Abc;_
2)sir lines ko constants.py me rakhne ki kya zaroort hai..?
   _get_ratings(..., lines=20)_
aisa kr du..?

Reply:
```
params = {}
params['param1'] = 'value'
params['param2'] = 'value2'
request(session, 'GET', url, params=params)
```

Reply:
This constant may be edited in the future to have a different value, that is why to keep it in constants.

Reply:
sir ye to pata hai...but url dekho...filterby wala param..usko kaise handle karu aise..?

Reply:
`{'filterBy': 'Country=india;Institution=Indian Institute of Information Technology, Design and Manufacturing, Jabalpur', 'itemsPerPage': 3, 'order': 'asc', 'sortBy': 'global_rank'}`
These params work perfectly.